.github/workflows/feature-request.yml

```diff
@@ -0,0 +1,38 @@
+name: Feature Request Manager
+on:
+  schedule:
+    - cron: 40 11 * * * # Run at 11:40 AM UTC (3:40 AM PST, 4:40 AM PDT)
+  issues:
+    types: [milestoned]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        if: github.event_name != 'issues' || contains(github.event.issue.labels.*.name, 'Feature Request')
+        uses: actions/checkout@v2
+        with:
+          repository: 'microsoft/vscode-github-triage-actions'
+          path: ./actions
+          ref: v35
+      - name: Install Actions
+        if: github.event_name != 'issues' || contains(github.event.issue.labels.*.name, 'Feature Request')
+        run: npm install --production --prefix ./actions
+      - name: Run Feature Request Manager
+        if: github.event_name != 'issues' || contains(github.event.issue.labels.*.name, 'Feature Request')
+        uses: ./actions/feature-request
+        with:
+          candidateMilestoneID: 30
+          candidateMilestoneName: Triage
+          backlogMilestoneID: 28
+          featureRequestLabel: Feature Request
+          upvotesRequired: 2
+          numCommentsOverride: 20
+          initComment: "This feature request is now a candidate for our backlog. The community has 60 days to upvote the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it.\n\nHappy Coding!"
```

Comment:
Does this comment match the policy?  I didn't see where in the action 20 upvotes will move it to the Backlog.  Do we need another bot for that or are we planning to track them and do this ourselves?  20 upvotes might be high for our repository since we have fewer users than VS Code all up.  Were we planning on moving all feature requests with 2 upvotes to the Backlog?